ports/vcpkg-make/vcpkg_make_install.cmake

```
@@ -0,0 +1,136 @@
+include_guard(GLOBAL)
+include("${CMAKE_CURRENT_LIST_DIR}/vcpkg_make.cmake")
+
+function(vcpkg_make_install)
+# Replacement for vcpkg_(install|build)_make
+    cmake_parse_arguments(PARSE_ARGV 0 arg
+        "DISABLE_PARALLEL"
+        "LOGFILE_ROOT;MAKEFILE;TARGETS"
+        "OPTIONS;OPTIONS_DEBUG;OPTIONS_RELEASE"
+    )
+    z_vcpkg_unparsed_args(FATAL_ERROR)
+
+    if(NOT DEFINED arg_LOGFILE_ROOT)
+        set(arg_LOGFILE_ROOT "make")
+    endif()
+
+    if(NOT DEFINED arg_TARGETS)
+        set(arg_TARGETS "all;install")
+    endif()
+
+    if (NOT DEFINED arg_MAKEFILE)
+        set(arg_MAKEFILE Makefile)
+    endif()
+
+    # Can be set in the triplet to append options for configure
+    if(DEFINED VCPKG_MAKE_OPTIONS)
+        list(APPEND arg_OPTIONS ${VCPKG_MAKE_OPTIONS})
+    endif()
+    if(DEFINED VCPKG_MAKE_OPTIONS_RELEASE)
+        list(APPEND arg_OPTIONS_RELEASE ${VCPKG_MAKE_OPTIONS_RELEASE})
+    endif()
+    if(DEFINED VCPKG_MAKE_OPTIONS_DEBUG)
+        list(APPEND arg_OPTIONS_DEBUG ${VCPKG_MAKE_OPTIONS_DEBUG})
+    endif()
+
+    if(CMAKE_HOST_WIN32)
+        set(Z_VCPKG_INSTALLED "${CURRENT_INSTALLED_DIR}")
+    else()
+        string(REPLACE " " "\ " Z_VCPKG_INSTALLED "${CURRENT_INSTALLED_DIR}")
+    endif()
+
+    vcpkg_make_get_shell(shell_var)
+    set(shell_cmd "${shell_var}")
+
+    find_program(Z_VCPKG_MAKE NAMES make gmake NAMES_PER_DIR REQUIRED)
+    set(make_command "${Z_VCPKG_MAKE}")
+
+    set(destdir "${CURRENT_PACKAGES_DIR}")
+    if (CMAKE_HOST_WIN32)
+        set(path_backup "$ENV{PATH}")
+        vcpkg_add_to_path(PREPEND "${CURRENT_HOST_INSTALLED_DIR}/share/vcpkg-make/wrappers") # This probably doesn't hurt but should it be guarded?

```

```suggestion
        vcpkg_add_to_path(PREPEND "${CURRENT_HOST_INSTALLED_DIR}/share/vcpkg-make/wrappers")
```

It seems like any open questions like this should have answers before we merge this.